src/extension/index.activateGithubAnalyses.ts

```diff
@@ -69,13 +69,25 @@ export function getPrimaryRepository(git: API): Repository | undefined {
 // githubCodeScanningInitialAlert.
 type ConnectToGithubCodeScanning = 'off' | 'on' | 'prompt' | 'injected' | undefined;
 
+export type GitHubCodeScanningEndpoint = 'https://api.github.com';
+
 export function activateGithubAnalyses(disposables: Disposable[], store: Store, panel: Panel, outputChannel: OutputChannel) {
     disposables.push(workspace.onDidChangeConfiguration(e => {
+        if (e.affectsConfiguration('sarif-viewer.githubCodeScanningEndpoint')) {
+            const gitHubCodeScanningEndpoint = workspace.getConfiguration('sarif-viewer').get<GitHubCodeScanningEndpoint>('githubCodeScanningEndpoint');
```

Comment:
Can we just use `string` as the type? The default API constant isn't strictly correct here because this setting can contain arbitrary strings.